packages/amplify-e2e-tests/src/__tests__/codegen/add_codegen.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable spellcheck/spell-checker */
```

Comment:
nit: eslint disable comments for new files should be avoided

We can add a missing word to our dictionary with `yarn addwords`, or if there's a string that doesn't makes sense to add, disable a specific line only.

Reply:
Ahh, my bad. I'd added that during dev so I wasn't messing w/ the file until I was done, forgot to remove it.